src/main/java/com/github/maven_nar/Msvc.java

```diff
@@ -164,6 +170,12 @@ public void configureLinker(final LinkerDef linker) throws MojoExecutionExceptio
         for (File sdkLib : sdkLibs)
           linker.addLibraryDirectory(sdkLib, sdkArch);
 
+      final String envLib = System.getenv("LIB");
```

Comment:
Perhaps then, the plugin should check whether the variables are defined in the environment in which `mvn` command is executed and pass these variables into the environment in which `cl` or `link` of cpp or link task are executed.  Currently, these variables are ignored and paths to header files and libraries are (somewhat arbitrarily) guessed and passed as command line arguments in the corresponding commands.  The assumption is that if `INCLUDE` or `LIBPATH` are set, they are set by running one of the VS `vars` scripts and define an environment for specific kind of build, be it a 32 or 64-bit target or cross-compilation build.